cmd/fpga_crihook/main.go

```diff
@@ -228,7 +227,7 @@ func getStdin(reader io.Reader) (*Stdin, error) {
 	return &stdinJ, nil
 }
 
-func (he *hookEnv) process(reader io.Reader) error {
+func (he *hookEnv) process(reader io.Reader, newPort newPortFun) error {
```

Comment:
Probably this dependency injection could be made only once in `newHookEnv()` to unify it with injecting `utilsexec.Interface`?

Reply:
thank you for the review! done.